vendor/github.com/giantswarm/k8scloudconfig/templates.go

```diff
@@ -514,8 +516,8 @@ coreos:
     content: |
       [Unit]
       Description=etcd2
-      Requires=k8s-setup-network-env.service
-      After=k8s-setup-network-env.service
+      Requires=k8s-setup-network-env.service decrypt-tls-assets.service
+      After=k8s-setup-network-env.service decrypt-tls-assets.service
```

Comment:
Yep, this is just a temporary hack because for some reason my `decrypt-tls-assets.service` wasn't starting.

It's not the proper way to solve it but I'll look at it tomorrow.